### PR TITLE
Replaced many "it" with better words

### PR DESCRIPTION
**Pull request by jsgro**
As a follow-up to issue #310 I replaced many "it" to clarify text after being invited to do so.

### CODE REVIEW

File: _episodes/05-writing-scripts.md [hunk 1]
```diff
@@ -60,10 +60,10 @@ The text at the bottom of the screen shows the keyboard shortcuts for performing
 
 > ## Which Editor?
 >
-> When we say, "`nano` is a text editor," we really do mean "text": it can
+> When we say, "`nano` is a text editor," we really do mean "text": `nano` can
```
Comment (akshayparopkari) [inline _episodes/05-writing-scripts.md:63]: Maybe `it` is more appropriate here, since the sentence already establishes `nano` as the subject.

**Comment by akshayparopkari** [comment]
Thank you, @jsgro! Congrats on your first PR 🎊 

Added in my suggestion to your edits. Everything else looks good to me 👍. I will leave this PR open for a few days to allow others time to comment on it.

What do you, @datacarpentry/shell-genomics-maintainers?

**Comment by aschuerch** [comment]
LGTM, thank you @jsgro and @akshayparopkari

**Comment by aschuerch** [comment]
hi @akshayparopkari, are you okay with closing this PR?

**Comment by akshayparopkari** [comment]
Looks good to me 👍

**Pull request merged**